README.md

```diff
@@ -40,78 +40,15 @@ Besides functionality to perform a typical recommend-measure loop, BayBE's highl
 - Fully typed and hypothesis-tested: Robust code base
 - All objects are fully de-/serializable: Useful for storing results in databases or use in wrappers like APIs
 
-## Installation
-### From Package Index
-The easiest way to install BayBE is via PyPI:
-
-```bash
-pip install baybe
-```
-
-A certain released version of the package can installed by specifying the
-corresponding version tag in the form `baybe==x.y.z`.
-
-### From GitHub
-If you need finer control and would like to install a specific commit that has not been
-released under a certain version tag, you can do so by installing BayBE directly from
-GitHub via git and specifying the corresponding 
-[git ref](https://pip.pypa.io/en/stable/topics/vcs-support/#git). 
-
-For instance, to install the latest commit of the main branch, run:
 
-```bash
-pip install git+https://github.com/emdgroup/baybe.git@main
-```
-
-
-### From Local Clone
-
-Alternatively, you can install the package from your own local copy.
-First, clone the repository, navigate to the repository root folder, check out the
-desired commit, and run:
-
-```bash
-pip install .
-```
-
-A developer would typically also install the package in editable mode ('-e'),
-which ensures that changes to the code do not require a reinstallation.
+## Quick Start
 
-```bash
-pip install -e .
-```
+First, install the `baybe` PyPI package with the `chem` and `simulation` dependency groups. For additional details, see the [advanced installation instructions](#advanced-installation).

```

Comment:
When building this locally, the link created here brought me to scikit-learn: https://scikit-learn.org/stable/developers/advanced_installation.html#advanced-installation

Reply:
Fixed, by providing an explicit anchor (+ also changing the name of the section)